src/PiWeb.Sdk.Import/build.targets

```diff
@@ -14,14 +19,26 @@
 
   <!-- This target can be manually called to pack the project as .pip package file -->
   <Target Name="PackPlugin" 
-          DependsOnTargets="_GeneratePluginPackage">
+          DependsOnTargets="Build;_GeneratePluginPackage">
   </Target>
 
   <!-- Creates a .pip package file for the project -->
   <Target Name="_GeneratePluginPackage" 
           DependsOnTargets="Build">
+
+    <ItemGroup>
+      <PipFiles Exclude="$(OutDir)\*.pip" Include="$(OutDir)\**\*" />
+    </ItemGroup>
+
+    <Message Text="Removing previously created staging directory."
+             Importance="high" />
+    <RemoveDir Directories="$(IntermediateOutputPath)\PipStaging" />
+
+    <Message Text="Copying build output to '$(IntermediateOutputPath)\PipStaging'."
+             Importance="high" />

```

Comment:
I would not set a high importance for the copy message.

Reply:
I set it to 'low'.